z3c/dependencychecker/dependencychecker.py

```diff
@@ -655,6 +655,9 @@ def set_log_level(verbose):
 
 
 def main():
```

Comment:
You *could* just move this to `main.py`?

Reply:
Yeah, that's my idea (and plenty more actually) I just wanted to have a minimal version to show to you first.